sharding-sql-test/src/main/resources/sql/supported/ddl/alter.xml

```diff
@@ -25,16 +25,17 @@
     <sql-case id="alter_table_with_back_quota" value="ALTER TABLE `t_order` FORCE" db-types="MySQL" />
     <sql-case id="alter_table_add_column" value="ALTER TABLE t_order ADD column4 VARCHAR(10)" />
     <sql-case id="alter_table_add_columns" value="ALTER TABLE t_order ADD column4 VARCHAR(10), ADD column5 VARCHAR(10), ADD column6 VARCHAR(10)" db-types="MySQL,PostgreSQL" />
-    <sql-case id="alter_table_add_columns_integer_type_mysql" value="ALTER TABLE t_order ADD column4 TINYINT, ADD column5 MEDIUMINT" db-types="MySQL" />
+    <sql-case id="alter_table_add_columns_integer_type_mysql" value="ALTER TABLE t_order ADD column4 INTEGER, ADD column5 TINYINT, ADD column6 MEDIUMINT" db-types="MySQL" />
```

Comment:
If the scope of this pr is for data type of SQLServer only, why do change test case for MySQL?

Reply:
Because I changed the test case `alter_table_add_columns_integer_type` to support `MySQL, PostgreSQL, SQLServer`, but SQLServer does not support Integer, so I modified  `MySQL`